python/lsst/sims/photUtils/Sed.py

```diff
@@ -995,23 +1022,26 @@ def calcNonSourceNoiseSq(self, skySed, hardwarebandpass, readnoise, darkcurrent,
         #This method outputs all of the parameters calculated along the way
         #so that the verbose version of calcSNR_psf still works
 
+        if photParams is None:
+            raise RuntimeError("Need to pass an instantiation of PhotometricParameters to " +
+                               "calcNonSourceNoiseSq")
+
         #Calculate the effective number of pixels for double-Gaussian PSF
-        neff = self.calcNeff(seeing, platescale)
+        neff = self.calcNeff(seeing, photParams.platescale)
```

Comment:
I don't understand why the seeing doesn't have a default value though (both here and in the Neff method call itself), given that you set the _seeingDefault values in the init. It seems inconsistent - either don't set it in the init, or use those defaults when calculating neff. Same with the platescale in calcNeff method. 
(I guess what I am arguing for is to make the seeing and platescale arguments in calcNeff to be kwargs, set to the class's default seeing and platescale values, and then let the user override when calling the various calc*Noise and calcSNR_PSF methods, but that those would also have the default values).